README.md

```diff
@@ -142,71 +119,78 @@ app.on('export-error', () => {})
 app.on('renderer-error', () => {})
 ```
 
-and launch it with `electron main.js`.
+…which can be launched with `electron main.js`.
 
 ## CLI
 
 See `./bin/`.
 
-### Graph exporter
+### Graph Exporter
 
-A specialised `runner` for plotly graphs:
+A specialized "runner" for Plotly graphs:
 
-```
+```sh
 plotly-graph-exporter 20.json https://plot.ly/~empet/14324.json --format svg
 ```
 
-where `20.json` is a local `"data"/"layout"` JSON file and
-`https://plot.ly/~empet/14324` is a JSON URL.
-
-I'm thinking the R, Python and Julia libraries could simply call
-`plotly-graph-exporter` to offer offline image generation to their users.
+…where `20.json` is a local `"data"/"layout"` JSON file and `https://plot.ly/~empet/14324` is the URL of a remote JSON resource.
 
-### Export server
+#### Python Usage Example
 
-Export server very similar to the current image server but with support
-for multiple components:
+```python
+from subprocess import call
+import json
 
+fig = {"data": [{"y": [1,2,1]}]}
+call(['plotly-graph-exporter', json.dumps(fig)])
 ```
-plotly-image-server --port 9090
 
-# Dispatch to one component based on url of request
-curl localhost:9090/plotly-graph/ <payload>
+#### R Usage Example
 
-# or e.g.
-curl localhost:9090/plotly-dashboard/ <payload>
+```R
+library(plotly)
+system2("plotly-graph-exporter", plotly_json(fig, FALSE))
 ```
```

Comment:
Unfortunately, it won't be this simple. Eventually we should provide a wrapper in the R package for this, but for now, you could use this R code:

```r
library(plotly)
p <- plot_ly(x = 1:10, y = 1:10, color = 1:10)
b <- plotly_build(p)$x[c("data", "layout")]
json <- plotly:::to_JSON(b)
cmd <- sprintf("plotly-graph-exporter '%s' -o r-export-test.png", json)
system(cmd)
```

Reply:
Thanks for the tip!

Reply:
done in https://github.com/plotly/image-exporter/blob/b8d52dc502c56dceba6b31fd57be47b9c2e3fd2b/README.md